frontend/public/assets/css/kiosk.css

```
@@ -139,6 +140,10 @@
      -moz-user-select: none;
       -ms-user-select: none;
           user-select: none;
+  text-rendering: optimizeLegibility;
+  font-smoothing: always;
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)